img/posts/2021-07-ln-linkability1.dot

```diff
@@ -0,0 +1,38 @@
+digraph ln_linky {
```

Comment:
Doesn't really matter at all, but these files could be named 2021-08-...

Reply:
I knew someone was going to nit on this.  :-)  I'm going to keep the names based on creation date since my main motivation for including dates in the file names is so I can find an illustration I remember creating x months/years ago.  I don't remember as well when we published things.

Reply:
nitters gonna nit